v2/concordium/types.proto

```diff
@@ -1945,6 +1945,14 @@ message ConsensusInfo {
   BlockHash current_era_genesis_block = 29;
   // Time when the current era started.
   Timestamp current_era_genesis_time = 30;
+  // The current duration to wait before a round times out.
+  optional Duration current_timeout_duration = 31;
+  // The current round.
+  optional uint64 current_round = 32;
+  // The current epoch.
+  optional uint64 current_epoch = 33;
+  // The trigger block time of the seedstate of the last finalized block.
```

Comment:
Maybe we could add that this is the nominal epoch boundary, and hence a block after this timestamp is considered the trigger block for the epoch. 

I don't think we should expose terms as the seed state in the API.